src/land/ghcn_snod2ioda.py

```diff
@@ -146,29 +158,32 @@ def assignValue(colrowValue, df400):
             lons = lons[mask]
             lats = lats[mask]
             alts = alts[mask]
+            sites = sites[mask]
             times = times[mask]
 
         # get datetime from input
         my_date = datetime.strptime(startdate, "%Y%m%d")
         start_datetime = my_date.strftime('%Y-%m-%d')
-        base_datetime = start_datetime + 'T12:00:00Z'
+        base_datetime = start_datetime + 'T18:00:00Z'
         AttrData['date_time_string'] = base_datetime
 
         for i in range(len(vals)):
             if vals[i] >= 0.0:
-                errs[i] = 0.0
-                vals[i] = 0.001*vals[i]  # coverted to m from mm
+                errs[i] = 40.0
+                vals[i] = 1.0*vals[i]  # use 1.0 for mm(use 0.001 for m)
             times[i] = base_datetime
-        self.loc_mdata['datetime'] = self.writer.FillNcVector(times, "datetime")
-        self.loc_mdata['latitude'] = lats
-        self.loc_mdata['longitude'] = lons
-        self.loc_mdata['altitude'] = alts
-        for iodavar in ['snowDepth']:
+        # add metadata variables
+        self.outdata[('datetime', 'MetaData')] = times
+        self.outdata[('stationIdentification', 'MetaData')] = sites
+        self.outdata[('latitude', 'MetaData')] = lats
+        self.outdata[('longitude', 'MetaData')] = lons
+        self.outdata[('altitude', 'MetaData')] = alts
```

Comment:
Similar to your other PR, you could now switch altitude to height in the needed places in code.

Reply:
Changed